content/en/docs/upgrade/devops-data-migration-upgrade.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "DevOps Data Migration during Upgrade"
```

Comment:
I assume we need to use `KubeSphere DevOps` in the title.

Reply:
Sure. It is better to use `KubeSphere DevOps`. Wording updated.